### PR TITLE
ci: add certificate to IDP and update docker

### LINKED ISSUE #21: fix: Sonarscanner did not collect IDP code

### PR DESCRIPTION
**Pull request by zovippro1996**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,14 +7,16 @@ COPY ./src/Planthor.IdentityServerAspNetIdentity/*.csproj ./src/Planthor.Identit
 RUN dotnet restore
 
 # copy everything else and run release app
-COPY ./src/Planthor.IdentityServerAspNetIdentity/. /app/Planthor.IdentityServerAspNetIdentity/
-WORKDIR /app/Planthor.IdentityServerAspNetIdentity
+COPY ./src/Planthor.IdentityServerAspNetIdentity/. /app/src/Planthor.IdentityServerAspNetIdentity/
+WORKDIR /app/src/Planthor.IdentityServerAspNetIdentity
```
Comment (akakshuki) [inline Dockerfile:11]: Should we add a sub-path `src` to planthor, In the future we will add another module to this container?

File: scripts/generate-dev-certs-powershell.sh
Comment (akakshuki) [inline scripts/generate-dev-certs-powershell.sh:1]: I have a concern... is this powershell syntax? Will we install pwsh cli to our Linux terminal ?

File: Dockerfile (hunk 1 above)
Comment (zovippro1996) [reply Dockerfile:11]: yes, we will have more project here (extensions + test)

File: scripts/generate-dev-certs-powershell.sh
Comment (zovippro1996) [reply scripts/generate-dev-certs-powershell.sh:1]: rename to ps1 powershell syntax (we should have a sh file doing similar later)

File: Dockerfile (hunk 1 above)
Comment (akakshuki) [reply Dockerfile:11]: hmm, I am not 100% sure that we need extensions + test submodule in the release stage... 🤔. But okay with me

File: scripts/generate-dev-certs-powershell.sh
Comment (zovippro1996) [reply scripts/generate-dev-certs-powershell.sh:1]: we don't have to install pwsh cli for now, but we need to have sh script if we want to have self-sign certificate.

**Review by akakshuki — APPROVED** [review]

**Pull request merged**